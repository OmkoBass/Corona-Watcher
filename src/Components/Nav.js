import React from 'react'


function Nav() {
    return (
        <nav className='text-center text-white p-3'
            style={{backgroundColor: 'black'}}
        >
            <h3>Corona Watcher</h3>
        </nav>
    )
}

export default Nav;
