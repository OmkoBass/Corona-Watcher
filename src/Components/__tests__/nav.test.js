import React from "react";

import { render, screen, cleanup } from "@testing-library/react";

import renderer from "react-test-renderer";

import Nav from "../Nav";

afterEach(() => {
  cleanup();
});

it("Should render Nav", () => {
  render(<Nav />);

  const navElement = screen.getByTestId("nav");

  expect(navElement).toBeInTheDocument();
});

it("Should contain text-center class", () => {
  render(<Nav />);

  const navElement = screen.getByTestId("nav");

  expect(navElement).toHaveClass("text-center text-white p-3");
});

it("Should render correctly", () => {
  const tree = renderer.create(<Nav />).toJSON();

  expect(tree).toMatchSnapshot();
});
