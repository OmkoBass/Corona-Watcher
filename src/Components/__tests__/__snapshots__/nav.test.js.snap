// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<nav
  className="text-center text-white p-3"
  data-testid="nav"
  style={
    Object {
      "backgroundColor": "black",
    }
  }
>
  <h3>
    Corona Watcher
  </h3>
</nav>
`;
